backend/tet/events/services.py

```diff
@@ -50,6 +50,7 @@ def _is_city_employee(self, user):
 
     def _get_event_and_raise_for_unauthorized(self, user, event_id):
         event = self.client.get_event(event_id)
+        LOGGER.warning(event)
```

Comment:
The logging is to Sentry, right? What does the event here contain? Might it contain sensitive/personally identifiable information which falls under GDPR?